bedita-app/models/behaviors/cacheable.php

```diff
@@ -112,6 +112,10 @@ public function getObjectsToCleanById(&$model, $objectId, array $excludeIds = ar
             'fields' => array('parent_id'),
             'conditions' => $treeConditions
         ));
+        // handle first save in tree (object still not in tree, but parent is in destination data)
+        if (!empty($model->data['BEObject']['destination']) && !in_array($model->data['BEObject']['destination'], $excludeIds)) {
```

Comment:
:thinking: `in_array($model->data['BEObject']['destination'], $excludeIds)` seems incorrect since `$model->data['BEObject']['destination']` should be an array and the operation results `true` only if all destinations are present in one value of `$excludeIds`. 

I think it's more safe to do
```php
if (!empty($model->data['BEObject']['destination'])) {
    $addParents = array_diff((array)$model->data['BEObject']['destination'], $excludeIds);
    $parents = array_unique(array_merge($parents, $addParents));
}
```